src/rogue.behavior.js

```diff
@@ -59,6 +59,8 @@ export default System.be({
       }
 
       return [
+        { Disassociate: [self, 'playerX', playerX] },
+        { Disassociate: [self, 'playerY', playerY] },
```

Comment:
Same issue as with the other example, provided detailed explanation in the comment below